src/styles/button.scss

```diff
@@ -0,0 +1,21 @@
+@import 'variables';
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  font-size: 14px;
+  border: none;
+  color: $color-white;
+  background-color: $color-button;
+  font-family: $font-roboto;
+  font-weight: $font-bold;
+  line-height: 16px;
+  margin: 0 16px 16px;
+
+  &:hover {
+    background-color: $color-white;
+    border: 1px solid $color-button;
```

Comment:
When you change the border-width on hover, it can cause the button to shift or resize, which might lead to layout shifts. To avoid this, use 'outline' or set a transparent border by default and change only the border-color on hover.